### PR TITLE
Feat/newsletter inputs

### PR DESCRIPTION
**Pull request by tomasmoose**
## Pivotal Tracker

https://app.asana.com/0/1189657201675175/1196980589573313

## Description

Include newsletter inputs for org,fname,lname when submitting a subscription + give subscription its own section on the landing page.

## Testing instructions

Ensure form works with new inputs, ensure subscription works with new inputs.

### CODE REVIEW

File: frontend/scripts/app/app.component.jsx [hunk 1]
```diff
@@ -41,6 +45,11 @@ function App() {
         <CookieBanner />
         {feedback && <Feedback />}
       </main>
+
+      <Suspense fallback={null}>
+        <NewsletterForm />
```
Comment (Bluesmile82) [inline frontend/scripts/app/app.component.jsx:50]: This is showing on every page. Even explore and tool. I think it's only supposed to go on the landing page. Isn't it?

File: frontend/scripts/react-components/shared/newsletter/newsletter.actions.js [hunk 2]
```diff
@@ -4,9 +4,12 @@ import axios from 'axios';
 export const NEWSLETTER__SET_SUBSCRIPTION_MESSAGE = 'NEWSLETTER__SET_SUBSCRIPTION_MESSAGE';
 export const NEWSLETTER__RESET_NEWSLETTER = 'NEWSLETTER__RESET_NEWSLETTER';
 
-export const sendSubscriptionEmail = email => dispatch => {
+export const sendSubscriptionEmail = ({ email, firstname, lastname, organisation }) => dispatch => {
```
Comment (Bluesmile82) [inline frontend/scripts/react-components/shared/newsletter/newsletter.actions.js:7]: I think we could do better error management (The errors are not showing correctly). And also some kind of success message. I'm not sure if this is part of this task but I don't think we will do it at any other time if we don't do it now

**Review by Bluesmile82 — CHANGES_REQUESTED** [review]
Please check the comments. I think it should only appear on the landing page

File: frontend/scripts/react-components/shared/newsletter/newsletter.actions.js (hunk 2 above)
Comment (tomasmoose) [reply frontend/scripts/react-components/shared/newsletter/newsletter.actions.js:7]: good point, talked with pablo and we will copy what they do on finance, stay tuned;

File: frontend/scripts/app/app.component.jsx (hunk 1 above)
Comment (tomasmoose) [reply frontend/scripts/app/app.component.jsx:50]: 🤦 good catch

Comment (tomasmoose) [reply frontend/scripts/app/app.component.jsx]: fixed

File: frontend/scripts/react-components/shared/newsletter/newsletter.actions.js (hunk 2 above)
Comment (tomasmoose) [reply frontend/scripts/react-components/shared/newsletter/newsletter.actions.js:7]: improved to a certain point!

**Comment by Bluesmile82** [comment]
I think the errors are not being cleared correctly. Instead of a confirmation message I get this:
image

**Pull request merged**